### PR TITLE
Conversion tool 1.0

### PR DESCRIPTION
**Pull request by Slickoo**
**Link** - https://github.com/orgs/NPlaylist/projects/1#card-19479594
**Title** - Conversion tool

## Description
Created console conversion application.

## How to use:
Type `./NPlaylist.Converter.exe -h` in cmd.exe

## Sanity Check

- [x] Update the change log above
- [x] Add at least two reviewers to the PR
- [ ] Implement new unit tests
- [x] Full run of solution unit tests

### CODE REVIEW

File: NPlaylist.Converter/ConversionTool.cs [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                case Format.PLS:
+                    return new AsxDeserializer();
+                default:
+                    return new M3uDeserializer();
+            }
+        }
+
+        private static string SerializePlaylist(IPlaylist playlist, Format fileFormat)
+        {
+            switch (fileFormat)
```
Comment (georgewwww) [inline NPlaylist.Converter/ConversionTool.cs]: You can add an
```csharp
default: return string.Empty();
```

File: NPlaylist.Converter/ConversionTool.cs [hunk 2]
```diff
@@ -0,0 +1,164 @@
+using System;
+using System.IO;
+using System.Linq;
+using NPlaylist.Asx;
+using NPlaylist.M3u;
+using NPlaylist.Wpl;
+using NPlaylist.Xspf;
+
+namespace NPlaylist.ReadingPlaylists
```
Comment (georgewwww) [inline NPlaylist.Converter/ConversionTool.cs]: It should be
```csharp
namespace NPlaylist.Converter
```
no?

File: NPlaylist.Converter/ConversionTool.cs [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+using NPlaylist.Wpl;
+using NPlaylist.Xspf;
+
+namespace NPlaylist.ReadingPlaylists
+{
+    public static class ConversionTool
+    {
+        public static void Convert(string initialPlaylistPath, string convertedPlaylistPath, Format outputFormat)
+        {
+            if (initialPlaylistPath == null)
```
Comment (georgewwww) [inline NPlaylist.Converter/ConversionTool.cs]: Better this
```csharp
String.IsNullOrWhiteSpace()
```

File: NPlaylist.Converter/Format.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+namespace NPlaylist.ReadingPlaylists
```
Comment (georgewwww) [inline NPlaylist.Converter/Format.cs]: Namespace pls

File: NPlaylist.Converter/ConversionTool.cs [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+            switch (fileFormat)
+            {
+                case Format.XSPF:
+                    return new XspfDeserializer();
+                case Format.WPL:
+                    return new WplDeserializer();
+                case Format.ASX:
+                    return new AsxDeserializer();
+                case Format.PLS:
+                    return new AsxDeserializer();
```
Comment (LopatchinIgor) [inline NPlaylist.Converter/ConversionTool.cs]: It should be PLS Deserializer

Comment (Slickoo) [reply NPlaylist.Converter/ConversionTool.cs]: Yeah,i didn't had Pls deserializer then

File: NPlaylist.Converter/ConversionTool.cs [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+                    return plsSerializer.Serialize(plsPlaylist);
+            }
+
+            return String.Empty;
+        }
+
+        private static Format GetFormatByExtension(string initialPlaylistPath)
+        {
+            var fileExtension = Path.GetExtension(initialPlaylistPath);
+            switch (fileExtension)
```
Comment (maksimenko-stanislav) [inline NPlaylist.Converter/ConversionTool.cs]: please call `ToLover()` on `fileExtension`.

**Pull request merged**